trade-finance-manager-ui/templates/utilisation-reports/_macros/search-reports-table.njk

```diff
@@ -0,0 +1,52 @@
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% import "./report-reconciliation-status.njk" as reconciliationStatus %}
+
+{% macro render(params) %}
+  {% set user = params.user %}
+  {% set reports = params.reports %}
+  {% set isTfmPaymentReconciliationFeatureFlagEnabled = params.isTfmPaymentReconciliationFeatureFlagEnabled %}
+  {% set reportRows = [] %}
+
+  {% for report in reports %}
+    {% set reportPeriodHtml =
+      '<p class="govuk-body govuk-!-margin-0">' + report.formattedReportPeriod + '</p>' 
+      if not isTfmPaymentReconciliationFeatureFlagEnabled 
+      else '<a class="govuk-link govuk-link--no-visited-state" href="/utilisation-reports/' + report.reportId + '">' + report.formattedReportPeriod + '</a>'
```

Comment:
Not keen on this syntax for concatenating strings, is there a better way to do it?

Reply:
In the `dashboard-deals-table.njk` they set a html fragment as follows:
```
{% set linkHtml %}
        <a
          href="{{ url }}{{ deal._id }}"
          class="govuk-link"
          data-cy="deal__link--{{ deal._id }}"
        >
          {{ deal.bankInternalRefName }}
        </a>
      {% endset %}

```
So perhaps could try something along those lines?

Reply:
Oooh yes, this is what I was looking for